README.md

```diff
@@ -0,0 +1,27 @@
+# Xedator
```

Comment:
Bruh

Reply:
shhhhhhhhhhhhhhhhhhhhhhhhhh you never saw it